### PR TITLE
Add template for postgresql with sample data

### PR DESCRIPTION
**Pull request by asmigala**

### CODE REVIEW

File: deploy/postgresql-with-data.yaml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    output:
+      to:
+        kind: ImageStreamTag
+        name: postgresql:latest
+    postCommit: {}
+    resources: {}
+    source:
+      contextDir: samples/postgresql
+      git:
+        uri: https://github.com/teiid/teiid-operator.git
```
Comment (asmigala) [inline deploy/postgresql-with-data.yaml]: The PR can be tested locally by chaning here to 
```
      git:
        ref: postgresql-with-data
        uri: https://github.com/asmigala/teiid-operator.git
```

**Comment by rareddy** [comment]
Although this pretty cool, there are a couple of reasons I do not want to accept it.
1) This is still template-based, I would rather base something on PG Operator based.
2) The README example to show the user how they quickly get their feet wet and test out the Operator than automating the install of the script with DDL, work like magic. The instructions IMO are simple enough. Frankly, there is a lot going on in there, not only it is installing the Postgres also starting a build that somehow reads this DDL file for schema creation.

I think this would an excellent blog article on how to set up your database with the initial state but needs to be changed using PG Operator, which will remove a lot of crud of Postgres stuff.

**Comment by asmigala** [comment]
@rareddy makes sense, I mostly did this to be able to quickly iterate with semi-manual testing of the operator, so since I already had this done, I decided to make a PR. (this is basically the output of the original `oc new-app` command with added `-o yaml`, plus using the upstream [s2i dump data example](https://github.com/sclorg/postgresql-container/tree/master/examples/s2i-dump-data)  )

But your reasoning makes sense, so closing the PR.

**Comment by rareddy** [comment]
That will still be good blog, I did not knew about until I went through your script.

**Pull request closed without merging**